docs/vectorizer/api-reference.md

```diff
@@ -111,14 +121,40 @@ SELECT ai.create_vectorizer(
 ```
 
 This function call:
-1. Sets up a vectorizer for the `website.blog` table.
-2. Loads the `contents` column.
-2. Uses the Ollama `nomic-embed-text` model to create 768 dimensional embeddings.
-3. Chunks the content into 128-character pieces with a 10-character overlap.
-4. Formats each chunk with a `title` and a `published` date.
-5. Grants necessary permissions to the roles `bob` and `alice`.
-
-The function returns an integer identifier for the vectorizer created, which you can use
+1. Sets up a vectorizer named 'website_blog_vectorizer' for the `website.blog` table.
+2. Creates a separate table `website.blog_embeddings_store` to store embeddings.
+3. Creates a view `website.blog_embeddings` joining the source and embeddings.
+4. Loads the `contents` column.
+5. Uses the Ollama `nomic-embed-text` model to create 768 dimensional embeddings.
+6. Chunks the content into 128-character pieces with a 10-character overlap.
+7. Formats each chunk with a `title` and a `published` date.
+8. Grants necessary permissions to the roles `bob` and `alice`.
+
+#### Example 2: Column destination
+
+For simpler cases where you need just one embedding per row (e.g. because your application already did the chunking), you can add a column directly to the source table:
```

Comment:
```suggestion
Column destination place the embedding in a separate column in the source table. It can only be used when the vectorizer does not perform chunking because it requires a one-to-one relationship between the source data and the embedding. This is useful in cases where you know the source text is short (as is common if the chunking has already been done upstream in your data pipeline).

The workflow is that your application inserts data into the table with a NULL in the embedding column. The vectorizer will then read the row, generate the embedding and update the row with the correct value in the embedding column.
```